Marlin/stepper.cpp

```diff
@@ -492,22 +474,32 @@ ISR(TIMER1_COMPA_vect) {
     // Check endstops
     if (check_endstops) {
       
-      #define _ENDSTOP(axis, minmax) axis ##_## minmax ##_endstop
+      #ifndef Z_DUAL_ENDSTOPS
```

Comment:
Everything is looking good. One point of style is that I've been using `#ifdef` first in cases like this, because it puts the "normal" code, which is usually smaller, closer to the code that follows. The use of `#ifndef` was designed so that the normal code comes first, which is not bad either. So, choose what makes more sense to you: normal closer to the following code or normal first.